validator/keymanager/v2/direct/backup.go

```diff
@@ -12,6 +12,25 @@ import (
 	keystorev4 "github.com/wealdtech/go-eth2-wallet-encryptor-keystorev4"
 )
 
+// ExportPrivateKeys retrieves the secret keys for the specified public
+// keys in the function input.
+func (dr *Keymanager) ExportPrivateKeys(
```

Comment:
Any tests for this new method?

Reply:
Where is this method used? Can we delete? Couldn't find any mentions of it

Reply:
That was leftover from an old implementation cause you originally wanted to have a subcommand rather than a flag to accounts-v2 list. I forgot to remove it :(